scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -208,14 +230,16 @@ filtered_sce
 
 ## Additional Quality control
 
-In addition to filtering out empty droplets, we also will want to filter out real cells that may have been damaged during library preparation.
+In addition to filtering out empty droplets, we also will want to filter out cells that may have been damaged during library preparation.
 These will often be characterized by a high proportion of mitochondrial transcripts.
-This is because, if a cell ruptures, the cytoplasmic transcripts will leak out, but mitochondrial transcripts, protected by the mitochondrial membrane, will be more protected.
-As a consequence, there is an over-abundance of mitochondrial reads.
+When a cell ruptures, cytoplasmic transcripts will leak out, but mitochondrial transcripts, still protected by the mitochondrial membrane, may remain.
+As a consequence, there will be an over-abundance of mitochondrial reads.
 
 Our first step then, is create a vector of the mitochondrial genes that are present in our dataset.
-The mitochondrial file we defined during setup (`mito_file`) is a TSV file with additional annotation information for each gene such as the gene location and alternative names. 
-All we need now is the `gene_id`, and only for the genes that are present in our SCE, so we will do some filtering with `dplyr` to pull out just a vector with those ids.
+The mitochondrial file we defined during setup (`mito_file`) is a TSV file containing all of the human mitochondrial genes with additional annotation information for each gene, such as the gene location and alternative names. 
+(For more detail on the steps we took to create this file, you can look at [one of our setup notebooks](https://github.com/AlexsLemonade/training-modules/blob/master/scRNA-seq-advanced/setup/mito_gene_lists.Rmd))
```

Comment:
Misc comment for whether this should be `master` or at a specific commit just in case?

Reply:
Ideally it is tagged to this release, but that is circular, so `master` should work for now. I don't expect this to move, but who knows. (Worth noting that we remove the setup notebooks when building user directories, so my initial idea of telling people to just look within their folders won't work.)